prog/vnet/load_balancer_nexus.rb

```diff
@@ -45,9 +48,27 @@ def before_run
 
     hop_create_new_health_probe if strand.children_dataset.count < load_balancer.vms_dataset.count
 
+    if load_balancer.need_certificates?
+      hop_create_new_cert
+    end
```

Comment:
nit: it will have the same syntax with the previous line `hop_create_new_health_probe if strand.children_dataset.count < load_balancer.vms_dataset.count`

```ruby
hop_create_new_cert if load_balancer.need_certificates?
```